app/src/main/java/com/example/plannr/admin/adminEdit/AdminEditFragment.java

```diff
@@ -55,38 +54,70 @@ public void onClick(View view) {
                 presenter.removeCourse();
             }
         });
+
+        EditText courseName = binding.adminEditNameField;
+        EditText courseCode = binding.adminEditCodeField;
+        CheckBox fall = binding.editFall;
+        CheckBox summer = binding.editSummer;
+        CheckBox winter = binding.editWinter;
+        EditText prereqs = binding.adminEditPrerequisiteField;
+
+        Course selected = CourseRepository.getSelectedCourse();
+
+        courseName.setText(selected.getCourseName());
+        courseCode.setText(selected.getCourseCode());
+        fall.setChecked(selected.getFallAvailability());
+        summer.setChecked(selected.getSummerAvailability());
+        winter.setChecked(selected.getWinterAvailability());
+
+        String temp = "";
+
+        if(!selected.getPrerequisites().isEmpty()) {
+            String[] prerequisites = selected.getPrerequisites().split(",");
+
+            for (int i=0; i<prerequisites.length; i++) {
+                if(!prerequisites[i].isEmpty()) {
+                    if(i > 0  && !prerequisites[i-1].isEmpty())
+                        temp += ",";
+                    temp += CourseRepository.getCourseById(Integer.parseInt(prerequisites[i])).getCourseCode();
+                }
+            }
+        }
+
+        prereqs.setText(temp);
```

Comment:
That's code for filling in the course edit form with the current information for ease of use.
For example, the course name TextView would say "CSCB07" instead of being blank when editing that course